src/api/Gemfile

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+
```

Comment:
I wonder why that is not done by default in the autocorrect of the `Style/FrozenStringLiteralComment`.... I'll open an issue in Rubocop :thinking: